firmware/config/engines/vw.cpp

```diff
@@ -19,6 +19,12 @@ void setVwAba() {
 	setFrankensoConfiguration();
 	engineConfiguration->camInputs[0] = Gpio::E0; // a random unused pin needed for happy HW CI
 
+	engineConfiguration->tps1_1AdcChannel = EFI_ADC_32;
```

Comment:
this block is confusing, please add code comment?

Reply:
added on [f4e001d](https://github.com/rusefi/rusefi/pull/7136/commits/f4e001d2946d5643bb368c296a03f6c0382ee91a)